api/app/reviews/tests.py

```diff
@@ -0,0 +1,441 @@
+        db.session.commit()
+
+        sections = [
+            Section(1, 'Tell Us a Bit About You', '', 1),
+            Section(2, 'Tell Us a Bit About You2', '', 2)
+        ]
+        db.session.add_all(sections)
+        db.session.commit()
+
+        questions = [
+            Question(1, 1, 'Why is attending the Deep Learning Indaba 2019 important to you?', 'Enter 50 to 150 words', 1, 'long_text', ''),
+            Question(1, 1, 'How will you share what you have learnt after the Indaba?', 'Enter 50 to 150 words', 2, 'long_text', ''),
+            Question(2, 2, 'Have you worked on a project that uses machine learning?', 'Enter 50 to 150 words', 1, 'long_text', ''),
+            Question(2, 2, 'Would you like to be considered for a travel award?', 'Enter 50 to 150 words', 2, 'long_text', '')
+        ]
+        db.session.add_all(questions)
+        db.session.commit()
+
+        closed_review = ReviewForm(2, datetime(2018, 4, 30))
+        closed_review.close()
+        review_forms = [
+            ReviewForm(1, datetime(2019, 4, 30)),
+            closed_review
+        ]
+        db.session.add_all(review_forms)
+        db.session.commit()
+
+        review_questions = [
+            ReviewQuestion(1, 1, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(1, 2, None, None, 'multi-choice', None, None, True, 2, None, None, 0),
+            ReviewQuestion(2, 3, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(2, 4, None, None, 'information', None, None, False, 2, None, None, 0)
+        ]
+        db.session.add_all(review_questions)
+        db.session.commit()
+
+    def get_auth_header_for(self, email):
+        body = {
+            'email': email,
+            'password': 'abc'
+        }
+        response = self.app.post('api/v1/authenticate', data=body)
+        data = json.loads(response.data)
+        header = {'Authorization': data['token']}
+        return header
+
+    def setup_one_reviewer_one_candidate(self):
+        responses = [
+            Response(1, 5, True)
+        ]
+        db.session.add_all(responses)
+        db.session.commit()
+
+        answers = [
+            Answer(1, 1, 'I will learn alot.'),
+            Answer(1, 2, 'I will share by doing talks.')
+        ]
+        db.session.add_all(answers)
+        db.session.commit()
+
+        response_reviewers = [
+            ResponseReviewer(1, 1) # assign reviewer 1 to candidate 1 response
+        ]
+        db.session.add_all(response_reviewers)
+        db.session.commit()
+
+    def test_one_reviewer_one_candidate(self):
+        self.seed_static_data()
+        self.setup_one_reviewer_one_candidate()
+        header = self.get_auth_header_for('r1@r.com')
+        params = {'event_id': 1}
+
+        response = self.app.get('/api/v1/review', headers=header, data=params)
+        data = json.loads(response.data)
+
+        assert data['reviews_remaining_count'] == 1

```

Comment:
It's better to use self.assertEqual() and other methods on the TestCase class, the advantage is that those methods have sensible error messages when the assertion fails.

Reply:
Done.

(When my tests were failing, I was wondering why it wasn't being helpful at all haha)